File: dataset/human/python/file_5861.py
def main():
    N = int(input())
    A = list(map(int, input().split()))
    B = list(map(int, input().split()))
    C = list(map(int, input().split()))
    p = -1
    r = 0
    for i in range(N):
        r += B[A[i]-1]
        if p + 1 == A[i]:
            r += C[p - 1]
        p = A[i]
    return r
print(main())

File: dataset/human/java/file_5861.java
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        HashMap<String, Integer> hashmap = new HashMap<>();
        String[] a = sc.next().split("");
        hashmap.put("a", 1);
        hashmap.put("b", 2);
        hashmap.put("c", 3);
        
        if (hashmap.get(a[0]) + hashmap.get(a[1]) + hashmap.get(a[2]) == 6) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}
